pallets/acurast/src/lib.rs

```diff
@@ -10,7 +10,7 @@ mod benchmarking;
 
 mod attestation;
 pub mod payments;
-mod traits;
+pub mod traits;
```

Comment:
@gitsimon why did you need to make it `pub`? everything is re-exposted publicly on line 21